pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -9,6 +9,35 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// BuildReason is a type used for populating the
+// Build Status.Reason field
+type BuildReason string
+
+const (
+	// SucceedStatus indicates that all validations Succeeded
```

Comment:
A nit comment, we should add a new line after each attribute, following the other attributes described in the same package.